src/main/java/bookstore/exception/CustomGlobalExceptionHandler.java

```diff
@@ -55,4 +55,13 @@ protected ResponseEntity<Object> handleEntityNotFoundException(EntityNotFoundExc
         body.put(MAP_ERROR_KEY, ex.getMessage());
         return new ResponseEntity<>(body, HttpStatus.NOT_FOUND);
     }
+
+    @ExceptionHandler({RegistrationException.class})
+    protected ResponseEntity<Object> handleRegistrationException(RegistrationException ex) {
+        Map<String, Object> body = new LinkedHashMap<>();
+        body.put(MAP_TIME_KEY, LocalDateTime.now());
+        body.put(MAP_STATUS_KEY, HttpStatus.NOT_FOUND);
+        body.put(MAP_ERROR_KEY, ex.getMessage());
```

Comment:
This code is duplicated with handleEntityNotFoundException method body.
Let's avoid it